src/pseudopeople/interface.py

```diff
@@ -150,8 +150,8 @@ def generate_decennial_census(
     Generates a pseudopeople decennial census dataset which represents simulated
     responses to the US Census Bureau's Census of Population and Housing.
 
-    :param source: The root directory containing pseudopeople input data. Defaults
-        to the pseudopeople sample datasets directory.
+    :param source: The root directory containing pseudopeople simulated population data. Defaults
+        to the pseudopeople sample population directory.
```

Comment:
I changed "sample datasets" to "sample population" to match the new terminology on the Simulated populations page, but the actual directory is still called `sample_datasets`. Perhaps we should change the directory name to `sample_population`.

Reply:
I think we don't consider this part of our public API.

Reply:
Nevertheless, I think it would be good to be consistent in our terminology. But since it's not public, we probably don't need to change it before yellow release.